src/SourceBuild/content/repo-projects/aspnetcore.proj

```diff
@@ -5,6 +5,7 @@
     <BuildScript>$(ProjectDirectory)eng\build$(ShellExtension)</BuildScript>
 
     <BuildActions Condition="'$(DotNetBuildSourceOnly)' != 'true'">$(FlagParameterPrefix)restore $(FlagParameterPrefix)all $(FlagParameterPrefix)pack $(FlagParameterPrefix)publish</BuildActions>
+    <BuildActions Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(Sign)' == 'true'">$(BuildActions) $(FlagParameterPrefix)sign</BuildActions>
```

Comment:
Is Sign ever true in source build only mode? Does this need the extra conditional?

Reply:
You're right, I don't think it'll ever be true. Given that assumption, I don't have a problem with removing the extra conditional.

Reply:
Removed in https://github.com/dotnet/sdk/pull/44855/commits/b8dc8d637b3a21baf61fec499e31ffd0bac09580